app/components/card.js

```diff
@@ -30,14 +30,11 @@ const Card = styled.article`
     & > p {
       margin-bottom: 0;
     }
-
-    & > div:not(.answer) {
-      display: none;
-    }
```

Comment:
I _believe_ the original purpose of this was to hide the resource summary on mobile(see [mobile designs](https://www.figma.com/file/UMxlpEWSlk9riKajOexvEe/SIAFG-%2F-Wireframes?node-id=353%3A0)).

When I implemented the react-markdown stuff, this clearly wouldn't work anymore anyway, but in either case, styling needs to be adjusted to hide those descriptions on mobile.

Reply:
Resolved in f5c12b8f8ac40d8cdeff482d21693abd5630e9e6 !